README.md

```diff
@@ -4,33 +4,76 @@ https://www.gro-intelligence.com/products/gro-api
 
 Client library for accessing Gro Intelligence's agricultural data platform.
 
-## Setup
+## Prerequisites
 
 1. [MacOS and Linux](unix-setup.md)
 2. [Windows](windows-setup.md)
 
-## Examples
+## Install Gro API client packages
 
-Download or clone this repository, and then you can navigate to our [api/client/samples/](api/client/samples/) folder and try executing the provided examples.
+There are two installation methods: using pip, or using git. 
 
-Try [quick_start.py](api/client/samples/quick_start.py)
+### Install with pip install (preferred)
 
 ```sh
-cd your-clone-path/api-client/api/client/samples/
+pip install git+https://github.com/gro-intelligence/api-client.git [--target=<your-gro-path>]
+```
 
-python quick_start.py
+If you don't specify your gro installation path, it will be the local [site-packages]( https://stackoverflow.com/questions/31384639/what-is-pythons-site-packages-directory) directory.
+
+### Install with git clone 
+
+```
+git clone https://github.com/gro-intelligence/api-client.git  [<your-gro-path>]
+```
+If you don't specify your gro installation path, it will be the current directory.
+In addition, optionally you can:
+
+1. [Set `PYTHONPATH=<your-gro-path>` as an environment variable] (https://www.techwalla.com/articles/how-to-set-your-python-path)
+
+2. Make `gro_client` an alias for `python <your-gro-path>/gro_client.py`
+
+
+## Gro API authentication token
+
+### Getting a token
+
+You can use the gro_client command line tool to request an authentication token. Note that you will be prompted to enter a password.
+
+```sh
+gro_client --user_email='email@example.com' --print_token
 ```
 
-Now should be able to find a sample output csv file at:
+This token is used throughout the Gro API client code, wherever authentication is required.
+
+### Saving your token as an environment variable
+
+If you don't want to enter a password or token each time, you can save
+the token as an environment variable. In the sample code, it is
+assumed that you have the token saved to your environment variables as
+GROAPI_TOKEN.
```

Comment:
Yeah, I was concerned that it would be too verbose for experienced users, and for novice users you end up having to assume shell/ IDE/OS version etc. or list every variant. Added some  links that do the job.